### DISCUSSION TITLE
Big initial overlaps next line

### DISCUSSION THREAD

**Discussion by Javallen** [sumatrapdfreader/sumatrapdf#4735] [Q&A]
Windows 10 64b , SumatraPDF ver.3.5.2 66b

In an epub: if an big initial is used when opening a chapter, the first line overlaps the second line.

Is this a problem from SumatraPDF or from the epub.

Jan
![Initial overlap](https://github.com/user-attachments/assets/c7549283-0a32-42c0-9ad3-ba6846fe13c6)

**Comment by GitHubRulesOK** [comment]
Unsure why it happens in some cases, but think MuPDF altered it to  be better since 3.5.2.
So check that same ePub out in pre-release https://www.sumatrapdfreader.org/prerelease and I think it will be handled correctly. if not then a sample file then would be needed.

**Comment by Javallen** [comment]
That's the solution!
![365 dagen jou - Blanka Lipińska (ver  3 5 2)](https://github.com/user-attachments/assets/3e43f1ad-862a-4414-93a6-1ecb423c2ebe)
![365 dagen jou - Blanka Lipińska (Prerelease)](https://github.com/user-attachments/assets/7e96eca7-4bca-4295-8751-eab72a730d94)
Thank you
